src/Query/Builder.php

```diff
@@ -720,17 +733,19 @@ public function from($collection, $as = null)
      */
     public function truncate(): bool
     {
-        $result = $this->collection->deleteMany([]);
+        $options = $this->inheritConnectionOptions();
+        $result = $this->collection->deleteMany($options);
```

Comment:
Subtle bug that would have gone unnoticed by type checking:

```suggestion
        $result = $this->collection->deleteMany([], $options);
```

Reply:
Good catch, updated.